RobotGain[] robot_gain_list
uint32 size
